src/main/java/seedu/address/logic/commands/person/AddCommand.java

```diff
@@ -34,7 +34,7 @@
             + PREFIX_DESCRIPTION + "End user";
 
     public static final String MESSAGE_SUCCESS = "New contact added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This contact already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This contact already exists in the contact list";
```

Comment:
Missing exclamation mark? (Just ensure the message punctuations are consistent throughout the models)